src/dual_model_variables.jl

```diff
@@ -0,0 +1,145 @@
+function add_dual_vars_in_dual_cones(dual_model::AbstractModel{T}, primal_model::AbstractModel{T}, 
+                                     con_types::Vector{Tuple{DataType, DataType}}) where T
+    primal_con_dual_var = Dict{CI, Vector{VI}}()
+    dual_obj_affine_terms = Dict{VI, T}()
+    for (F, S) in con_types
+        num_con_f_s = MOI.get(primal_model, MOI.NumberOfConstraints{F, S}()) # Number of constraints {F, S}
+        for con_id = 1:num_con_f_s
+            # Add dual variable to dual cone
+            # Fill a dual objective dictionary
+            # Fill the primal_con_dual_var dictionary
+            add_dual_variable(dual_model, primal_model, primal_con_dual_var, dual_obj_affine_terms, con_id, F, S)
+        end
+    end
+    return primal_con_dual_var, dual_obj_affine_terms
+end
+
+# Utils for add_dual_variable functions
+function push_to_dual_obj_aff_terms!(dual_obj_affine_terms::Dict{VI, T}, vi::VI, value::T) where T
+    return iszero(value) ? nothing : push!(dual_obj_affine_terms, vi => value)
```

Comment:
again i'd probably make this a multiline if statement

Reply:
same here